.github/CODEOWNERS

```diff
@@ -59,10 +59,10 @@ packages/messaging-interop-types @zwu52 @firebase/jssdk-global-approvers
 integration/messaging @zwu52 @firebase/jssdk-global-approvers
 
 # Auth Code
-packages/auth  @avolkovi @lisajian @sam-gc @yuchenshi @firebase/jssdk-global-approvers
-packages/auth-compat @avolkovi @lisajian @sam-gc @yuchenshi @firebase/jssdk-global-approvers
-packages/auth-types @avolkovi @lisajian @sam-gc @yuchenshi @firebase/jssdk-global-approvers
-packages/auth-interop-types @avolkovi @lisajian @sam-gc @yuchenshi @firebase/jssdk-global-approvers
+packages/auth  @chuanr @lisajian @prameshj @sam-gc @firebase/jssdk-global-approvers
+packages/auth-compat @chuanr @lisajian @prameshj @sam-gc @firebase/jssdk-global-approvers
+packages/auth-types @chuanr @lisajian @prameshj @sam-gc @firebase/jssdk-global-approvers
```

Comment:
Is @renkelvin the github username that you meant to include instead of chuanr?

Reply:
Yes, updated :) Thanks!